lib/diagnostic.js

```diff
@@ -46,7 +46,9 @@ let person2 = {
 };
 
 // Question 3:  Invoke the `fullName()` method so it returns `Santa Clause`?
-let answer3 = null; // replace null with your response
+let answer3 = person1.fullName().call(person2); // replace null with your response
```

Comment:
This will cause an error because `call()` is a method on `Function` and, through method chaining, `person1.fullName()` returns a string, which then attempts to invoke `call()`.

`person1`'s `fullName()` method needs to be called with `person2` as the context.  This is done by calling `Function`'s `call()` method on `fullName()` with `person2` as the argument.

```javascript
person1.fullName.call(person2);
```